db/test/unit/computed_columns/application_analyst_lead_test.sql

```diff
@@ -0,0 +1,22 @@
+begin;
+
+select plan(3);
+
```

Comment:
I think a test here that tests the return value would be useful, and at least one with multiple lead changes on the same application